draft-ietf-quic-multipath.md

```diff
@@ -1139,14 +920,11 @@ path identifier.
     the control frame.
     Note that this is the connection identifier used by the peer
     when sending packets on the to-be-closed path.
-    This method SHOULD be used if this connection identifier is non-zero
-    length. This method MUST NOT be used if this connection identifier
-    is zero-length.
+    This method SHOULD be used by default.
```

Comment:
We can of course delegate this on a subsequent PR, but I think we should get rid of the `Identifier Type` for the PATH_ABANDON and PATH_STATUS frames. Also, I wonder it the value 1 should rather be the default (given #137).

Reply:
Yes, I was also planning to raise a separate issue and PR on this one after the merge.